features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/stats/api-usage-user/js/stats.js

```diff
@@ -147,7 +147,7 @@ var drawAPIUsage = function (from,to) {
 
                              for (var i = 0; i < length; i++) {
 
-                                 var apiData= JSON.parse(json.usage[i].apiName);
+                                 var apiData= json.usage[i].apiName;
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea